prog/weaveutil/netdev.go

```diff
@@ -33,3 +94,39 @@ func configureARP(args []string) error {
 
 	return nil
 }
+
+// listNetDevs outputs network ifaces identified by the given indexes
+// in the format of weavenet.Dev.
+func listNetDevs(args []string) error {
```

Comment:
I think we should:
* Pass the indices as separate args so you can ditch the `Split`
* Don't bother de-duplicating - the list always comes from `ConnectedToBridgeVethPeerIds` anyway, which can't generate dupes
* Add annotations to the NetDev structure, and print as JSON - that'll fix your ParseNetDev issue at the same time